Packs/CommonScripts/Scripts/FormatURL/FormatURL.py

```diff
@@ -102,6 +102,18 @@ def replace_protocol(url_: str) -> str:
     return url_
 
 
+def remove_brackets_from_end_of_url(url_: str) -> str:
```

Comment:
why `url_` and not `url`?

Reply:
I followed the convention of the function above.